widgetastic_manageiq/__init__.py

```diff
@@ -4725,3 +4728,23 @@ def select_plan(self, plan_name):
         """Find item by text and click to view details."""
         el = self._get_card_element(plan_name)
         self.browser.click(self.TITLE_LOCATOR, parent=el)
+
+    def datastore_progress_percent(self, plan_name):
```

Comment:
Optional:
I  would like to go with a single method by iteration  over `.//div[@class="progress-bar"]`
append `status` for `vm` and `datastore` as a key of `progress` method.

Reply:
superb suggestion, working on it!